docs/UserGuide.md

```diff
@@ -229,7 +231,7 @@ A person can have any number of tags (including 0)
 * For example, for Robert Johnson (the example command), it would be:<br> `New person added: Robert Johnson; Phone: 55512345; Email: robertj@email.com; Occupation: Hairdresser; Address: 789 Oak Street, Suite 10; AppointmentDate: ; Tags: `<br> Please note that both Appointment Date and Tags are empty as they are not necessary for adding a person
```

Comment:
for some reason, it says Robert Johnson (the example command) when it wasnt mentioned prior. Should change the phrasing or add Robert Johnson to the example commands.